docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -33,14 +33,21 @@
 
 \section{Team Meeting Plan}
 
+\item Standup meetings are scheduled weekly from \textbf{12:30 PM to 1:30 PM every Wednesday}. The primary meeting location is the \emph{H.G. Thode Library of Science \& Engineering}. However, in specific circumstances and within certain limitations, the meeting location may be shifted to \emph{Discord}.
+    \item Sprint review meetings are scheduled weekly from \textbf{5:00 PM to 6:00 PM every Sunday}. The meeting location is \emph{Discord}.
+    \item The roles and agendas for the next meeting will be decided in the final minutes of each meeting.
+    \item Requests for additional meetings outside of regular scheduled meetings must be made and communicated at least \textbf{1 day in advance}.
+    \item The Chair of the meeting is responsible for preparing the task backlog and identifying any relevant issues prior to the meeting.
+\end{itemize}
+
 \section{Team Communication Plan}
 
-The primary means for communication will be the team \emph{Discord} server. In an effort to 
+The primary means for communication will be the team \emph{Discord} server. In an effort to
 ensure team members are up-to-date with messages and discussion, everyone is expected to check \emph{Discord}
 at least once per day and respond to direct messages within 24 hours.
-Additionally, the server has been organized in such a way that different topics are separated into different channels. 
-For general  discussion and planning, the \emph{general} channel will be used. When requesting reviewers on pull requests, 
-the \emph{pull-request} channel should be used. Any bugs and issues that require group consultation will go 
+Additionally, the server has been organized in such a way that different topics are separated into different channels.
+For general  discussion and planning, the \emph{general} channel will be used. When requesting reviewers on pull requests,
+the \emph{pull-request} channel should be used. Any bugs and issues that require group consultation will go
```

Comment:
Can you revert the changes to the Team Communication plan? I have a new PR that updates the entire section and I can just remove spaces in my own PR.

Reply:
I think this happened cuz I sourced the branch from either master that did not have the changes you made earlier.
I'll revert the Team Comm section.

Reply:
Done!